bench/pom.rb

```diff
@@ -1,55 +1,55 @@
-version = ENV['JRUBY_VERSION'] ||
-  File.read(File.join(basedir, '..', 'VERSION')).strip
+version = ENV["JRUBY_VERSION"] ||
+          File.read(File.join(basedir, "..", "VERSION")).strip
```

Comment:
really rubocop default configuration uses `'` ?
been operating for years based on (GH's?) Ruby guide to prefer `'` unless interpolation is used.
why would they diverge from a default Ruby style guide that has been around prior... :unamused: